cmd/base.go

```diff
@@ -169,6 +169,10 @@ func EnsureCLIFlags(cmd *cobra.Command, appcfg *bc.Config) {
 			elem = reflect.ValueOf(&appcfg.HARE).Elem()
 			assignFields(ff, elem, name)
 
+			ff = reflect.TypeOf(appcfg.HareEligibility)
```

Comment:
I don't see "GenesisActiveSet` as part of this PR title so I don't understand why is this a part of this pr.
also, you didn't add any of the other config code. (the actual config struct and config.toml and yaml files.)

Reply:
1. This PR uses params I added a few PRs ago so we can check non-genesis flow regarding the hare's oracle. When I added the config I missed adding the bug-patch, hence causing this test to fail so I fixed it in this PR.
2. There is no additional test. I just adjusted test_mining params as you can see from its config file.